air/src/options.rs

```diff
@@ -63,27 +39,23 @@ pub enum FieldExtension {
 /// These parameters have a direct impact on proof soundness, proof generation time, and proof
 /// size. Specifically:
 ///
-/// 1. Hash function - proof soundness is limited by the collision resistance of the hash function
-///    used by the protocol. For example, if a hash function with 128-bit collision resistance is
-///    used, soundness of a STARK proof cannot exceed 128 bits.
```

Comment:
I would probably not remove this as it is an import part of security level considerations. We can move this to the end, and make it clear that hash function is not a part of the options. Something along the lines: "Another important parameter in defining STARK security level, which is not a part of [ProofOptions] is a hash function used in the protocol...".

Reply:
Done